booking-engine-api/guidelines/authentication.md

```diff
@@ -0,0 +1,36 @@
+# Authentication
+
+The __Mews Booking Engine API__ is a public API that requires client authentication and authorization. 
+
+## Client authentication
+
+Authentication is the process of verifying the identity of the client. To access the API, you must identify your client application by providing the `Client` property in all requests. This ensures that your client application is recognized and allowed to interact with the API.
+
+### Pre-registered
```

Comment:
suggestion: Pre-registered... what? How about "Client registration" or "Client pre-registration"? Personally I don't think calling it pre-registration adds much value compared to just registration.

Reply:
Yes, quite right. I suggest changing the title to "Registration".